README.md

```diff
@@ -137,18 +200,50 @@ Replaces a named policy with one from the provided file. This is
 usually a destructive invocation. Result is a dictionary object
 constructed from the returned JSON data.
 
-#### `delete_policy_file(policy_name, policy_file)`
+#### `update_policy_file(policy_name, policy_file)`
 
 Modifies an existing Conjur policy. Data may be explicitly
-deleted using the !delete, !revoke, and !deny statements. Unlike
+deleted using the `!delete`, `!revoke`, and `!deny` statements. Unlike
 "replace" mode, no data is ever implicitly deleted. Result is a
 dictionary object constructed from the returned JSON data.
 
-#### `list()`
+#### `list(list_constraints)`
 
-Returns a Python list of all the available resources for the current
+Returns a list of all available resources for the current
 account.
 
+The list constraints parameter is optional and should be provided as
+a dictionary.
+
+For example: `client.list({'kind': 'user', 'inspect': True})`
+
+| List constraints | Explanation                                                  |
+| ---------------- | ------------------------------------------------------------ |
+| kind             | Filter resources by specified kind (user, host, layer, group, policy, variable, or webservice) |
+| limit            | Limit list of resources to specified number                  |
+| offset           | Skip specified number of resources                           |
+| role             | Retrieve list of resources that specified role is entitled to see (must specify role’s full ID) |
+| search           | Search for resources based on specified query                |
+| inspect          | List the metadata for resources                              |
+
+#### `rotate_other_api_key(resource: Resource)`
+
+Rotates another entity's API key and returns it as a string.
+
+Note: resource is of type Resource which should have `type` (user / host) and 
+`name` attributes.
+
+#### `rotate_personal_api_key(logged_in_user, current_password)`
+
+Rotates the personal API key of the logged-in user and returns it as a string.
+
+#### `change_personal_password(logged_in_user, current_password, new_password)`
+
+Updates the current, the logged in user's password with the password parameter provided. 
+
+Note: the new password must meet the Conjur password complexity constraints. It must 
+contain at least 12 characters: 2 uppercase, 2 lowercase, 1 digit, 1 special character.
+
 #### `whoami()`
 
 _Note: This method requires Conjur v1.9+_
```

Comment:
@sigalsax  - I don;t see a conjur oss version 1.9. Please see list of releases and consult with @izgeri

Reply:
@sigalsax  - what about this?
@izgeri ??

Reply:
it is 1.9
<img width="910" alt="Screen Shot 2021-04-12 at 9 14 32 AM" src="https://user-images.githubusercontent.com/19418506/114348830-c0028d80-9b6f-11eb-979e-707831271ceb.png">

Reply:
@InbalZilberman